installing/installing_azure/ipi/installing-azure-private.adoc

```diff
@@ -43,7 +43,13 @@ include::modules/installation-azure-arm-tested-machine-types.adoc[leveloffset=+2
 include::modules/installation-azure-trusted-launch.adoc[leveloffset=+2]
 include::modules/installation-azure-confidential-vms.adoc[leveloffset=+2]
 
-include::modules/installation-azure-config-yaml.adoc[leveloffset=+2]
+include::modules/installation-azure-config-yaml-simple.adoc[leveloffset=+2]
```

Comment:
Similar to https://github.com/openshift/openshift-docs/pull/92943/files#r2131397209, if we can have a dedicated section to guide users to set `publish: Internal` in install-config.yaml for a private cluster install. That would be helpful.